GoogleDataTransport/GDTLibrary/GDTPlatform.m

```diff
@@ -73,6 +73,20 @@ - (instancetype)init {
                            selector:@selector(iOSApplicationWillTerminate:)
                                name:name
                              object:nil];
+
+#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
Probably also want this for TARGET_OS_MACCATALYST

Reply:
I'd prefer to wait until I have Catalyst to test that. I'll send Catalyst changes in a separate PR.